app/controllers/api/v1/study_files_controller.rb

```diff
@@ -687,6 +687,7 @@ def study_file_params
                                            :description, :is_spatial, :file_type, :status, :human_fastq_url, :human_data, :use_metadata_convention,
                                            :cluster_type, :generation, :x_axis_label, :y_axis_label, :z_axis_label, :x_axis_min,
                                            :x_axis_max, :y_axis_min, :y_axis_max, :z_axis_min, :z_axis_max, :species, :assembly,
+                                           :external_link_url, :external_link_title, :external_link_description,
```

Comment:
BLOCKER: We're defining these parameters at the `StudyFile` level, but then they are also present in `ClusterFileInfo` as well (and apparently unused there).  We should settle on one over the other to avoid confusion.

My preference would be to put them in `ClusterFileInfo`, as that data is specific to cluster files, and we already have way too many misplaced attributes on `StudyFile` documents (like axis ranges, labels, etc).  I won't consider moving the data there to be a blocker - just the decision of one storage location over the other.

Passing the parameters through would be more challenging, though - you'd need to add a block like this to `study_file_params`:
```
cluster_file_info_attributes: { :external_link_url, :external_link_title, :external_link_description }
```
You would then need to add them to the correct form with the parameters namespaced accordingly so that they assign to the nested document (this may require a new component), and (possibly) add `study_file.build_cluster_file_info if study_file.file_type == 'Cluster` to `StudiesController#new_study_file`.